lib/matplotlib/axes/_axes.py

```diff
@@ -4568,6 +4568,12 @@ def scatter(self, x, y, s=None, c=None, marker=None, cmap=None, norm=None,
             or ``nan``). If ``True`` the points are drawn with the *bad*
             colormap color (see `.Colormap.set_bad`).
 
```

Comment:
I would not call this just scale, because it could be interpreted as "linear scaling factor" - and we do this e.g. at `mutation_scale` in https://matplotlib.org/devdocs/api/_as_gen/matplotlib.patches.FancyArrowPatch.html#matplotlib.patches.FancyArrowPatch. Suggestion: `markerscaling` or maybe just `scaling` - I have some sympathy to push this down to collectionWithSizes` later. A `scaling` parameter would be meaningful there so that we have consistent naming and don't need to translate the markerscaling kwarg. Also, I believe scatter(..., scaling=...) is clear enough.

Also, 1 and 2 are not very telling as values. How about linear/quadratic or linear/square or length/area?